### PR TITLE
HA: fix curl calls in systemd resource agent tests

### PR DESCRIPTION
**Pull request by lucaskanashiro**
Those changes were tested in `dugtrio` and also in my host running Focal, everything seems to be working fine for me.

### CODE REVIEW

File: ha/virsh/run_tests.sh [hunk 1]
```diff
@@ -9,10 +9,10 @@ source /etc/profile.d/libvirt-uri.sh
 
 test_failed=0
 for file in $TESTS; do
-  AGENT=$(echo "$file" | grep -oP '(?<=/).+(?=\_)')
+  AGENT=$(echo "$file" | grep -oP '(?<=/).+(?=\_)' | tr _ -)
```
Comment (paride) [inline ha/virsh/run_tests.sh:12]: Is this change relevant to this fix? More in general, why do we need it?

Comment (lucaskanashiro) [reply ha/virsh/run_tests.sh:12]: `hostname` command is removing the `_` which makes the teste fail. If we use only `-` this is not a problem.

**Review by paride — APPROVED** [review]
LGTM and works fine, thanks!
